gtd-timer/Controllers/SignUpController.cs

```diff
@@ -43,18 +45,19 @@ public IActionResult GetByID(int id)
         [HttpPost]
         public ActionResult Post(UserDTO model)
         {
+            PasswordHasher pshash = new PasswordHasher();
             var userExist = unitOfWork.Users.GetAll().Any(u => u.Email == model.Email);
             if (userExist)
             {
                 return BadRequest("User with such email address already exist");
-            } 
+            }
 
             User user = new User
             {
                 FirstName = model.FirstName,
                 LastName = model.LastName,
                 Email = model.Email,
-                PasswordHash=model.Password
+                PasswordHash = pshash.HashPassword(model.Password)
```

Comment:
as another agreement - don't push unrelated changes in one pull request. 
Pull requests and branches are cheap, and separating different changes saves a lot of headache in the future.

Reply:
Ok i will do that in future. Can you aprove it or i need to divide it on two different requests?